client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/hooks/useNextItem.tsx

```diff
@@ -1,6 +1,8 @@
 import { ItemRowFragment, toItemRow } from '@openmsupply-client/system';
 import { usePrescription } from '../../../api';
 
+// TODO: DELETE THIS FILE
```

Comment:
This isn't needed in the new design but didn't want to break it just yet as I still have the modal...
Might be other stuff to remove too, worth checking....